src/styles/globalStyles.ts

```diff
@@ -56,16 +57,26 @@ const globalStyles = css`
     @media screen and (max-width: 1024px) {
       --space-unit: 0.8em;
     }
+
+    /* Shadows */
+    --elevation-1: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
```

Comment:
I'd be curious of others opinions on this: it took me a few seconds to understand what an "elevation" css property was for - which would be a shadow that gives the appearance of a certain level of elevation.

Would it make more sense to just call these `shadow` for future ease of use? Or is elevation already an established convention, and I am just behind :sweat_smile:

Reply:
@nobrayner It's a convention I've seen elsewhere. Basically a higher elevation number should mean that the element appears to 'float' at a higher elevation off of the page/screen - I think it's originally a material design convention. Obviously it should be renamed if different elevations aren't part of the design.

Reply:
elevation is borrowed from [material design](https://material.io/design/environment/elevation.html), it's the most popular design methodology so it's safe to assume that most devs would recognise it. But it also gives space to other shadow-themed variables without confusion

Reply:
I've seen this convention as well, specifically from Material UI

Reply:
Cool - I learned something new :+1: